src/main/java/edu/kis/vh/nursery/defaultCountingOutRhymer.java

```diff
@@ -17,19 +20,19 @@ public void countIn(int in)
 
 	public boolean callCheck()
 	{
-		return total == -1;
+		return total == TOTAL;
```

Comment:
W klasie DefaultCountingOutRhymer stałe o wartości -1 odnoszące się do pola total (też w metodzie callCheck), to nie jest ta sama stała o wartości -1 oznaczające domyślną zwracaną przez wyliczankę wartość (w metodach peekaboo i countOut). 
sens stałej dotyczącej domyślnej zwracanej wartości można określić za pomocą słów, np. IF_EMPTY lub DEFAULT, RETURN, VALUE (np. DEFAULT_VALUE).